### PR TITLE
Add WebAssembly features

### PR DESCRIPTION
**Pull request by petele**
Worked with @tomayac our Wasm DA to break out the features into appropriate chunks.

### CODE REVIEW

File: features/wasm-multi-memory.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: WebAssembly multi-memory
+description: An extension to WebAssembly to all the use of multiple memories within a single Wasm module.
```
Comment (autonome) [inline features/wasm-multi-memory.yml:2]: s/all/allow/

**Comment by Elchi3** [comment]
@petele Are you planning to come back to this PR? I'm happy to take over otherwise.

**Comment by petele** [comment]
@Elchi3 I prob won't have time to pick it up again until late November, so if you want to grab it, please feel free. Sorry, and thank you!!

**Comment by Elchi3** [comment]
No worries Pete, I will try to drag this over the finish line in https://github.com/web-platform-dx/web-features/pull/1970.

**Pull request closed without merging**